### ISSUE TITLE
Deleted questions

### ISSUE THREAD

**Issue by Bjoern-Ge** [KQMATH/moodle-mod_capquiz#215]
When adding a question to a normal quiz, you cannot delete it fully.
When trying to delete it, you get a warning, that the questions used in a quiz.

It would be nice if moodle keeps a copy of everything added to a CAPQuiz and gives a warning if you try to delete it.

An alternative would be that CAPQuiz automatic removes deleted questions from the CAPQuiz.

**Comment by hgeorgsch** [comment]
I agree that this is an issue.  It should wait until we have something working for Moodle 4.

**Comment by hgeorgsch** [comment]
New discussion here: https://moodle.org/mod/forum/discuss.php?d=446228
This seems to be tricky to do, until a new API has been created for the purpose in qbank.

**Comment by sebastsg** [comment]
This feature was added already in current beta, but there was a typo introduced sometime.
Hijacking this issue to link to #252.
